editor/src/messages/portfolio/document/document_message_handler.rs

```diff
@@ -702,6 +702,8 @@ impl MessageHandler<DocumentMessage, DocumentMessageData<'_>> for DocumentMessag
 				};
 
 				let size = existing_bottom_right - existing_top_left;
+				// If size is zero we clamp it to minimun value to avoid dividing by zero vector to calculate enlargement.
```

Comment:
```suggestion
				// TODO: This is a hacky band-aid. It still results in the shape becoming zero-sized. Properly fix this using the correct math.
				// If size is zero we clamp it to minimun value to avoid dividing by zero vector to calculate enlargement.
```